### ISSUE TITLE
location of webpack output path/directory structure

### ISSUE THREAD

**Issue by BernardA** [Limenius/symfony-react-sandbox#42]
Hello
I am surely missing something, but I cannot find the output path stated in both webpack.config.js and webpack.config.serverside.js
The files are supposed to be at `web/build/` and `app/Resources/webpack/`respectively, but no such directories exist in the repository. 
Also, I am following the 'standard' webpack encore directory structure, with an `assets` and another  `node-modules` directories at the root of my project. This do no seem to be consistent with your setup, or is it?
Thanks

**Comment by nacmartin** [comment]
@BernardA what was happening? Can we add something to the docs to make it clearer to other users?

**Comment by BernardA** [comment]
@nacmartin, thanks for getting back to me.
Actually it's mostly lack of experience with github. I was expecting to see the directories already populated by webpack.
It would probably be helpful to inform that one should add Symfony's 'vendor' directory which is not included.
When I downloaded the repository I finally managed to get it to work. I still have to understand the whole thing and transplant it to my own project. Hopefully this will take hours and not days.
There's certainly lots to  be made so that the very good tool that are forms in Symfony could be more intuitively utilized together with front-end tools like React. I compliment you for the very good job in did in that direction.
